book/website/community-handbook/infrastructure/infrastructure-contributors.md

```diff
@@ -0,0 +1,42 @@
+# Contributors
```

Comment:
For clarity, it might be better to change this title to "Recording Contributors", or something along those lines? This might help to signpost the difference between the infrastructure required to record contributors, and actual process of recording them.